lib/rules/func-name-matching.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * @fileoverview Rule to require function names to match the name of the variable or property to which they are assigned.
+ * @author Annie Zhang,
```

Comment:
yes! that's why the comma's there actually, but i forgot to follow up & ask who i should credit